### ISSUE TITLE
Obsidian_to_Anki clearing fields not included in Obsidian note

### ISSUE THREAD

**Issue by anahodil** [Pseudonium/Obsidian_to_Anki#527]
Hello everyone,
I am using Obsidian_to_Anki to create notes for Italian. Then, once in Anki, I use a TTS tool to generate audio for the card. However, whenever I trigger a vault scan, the audio field in Anki is cleared. 

To say it a different way

Given that I have a note in Obsidian
And the note has been generated in Anki using Obsidian_to_Anki
And the note in Anki has a field not present in the Obsidian note
And the field in Anki is populated
Then when I scan the vault with Obsidian_to_Anki
I expect fields called out in Obsidian to be updated, but fields not specified in Obsidian to remain unchanged

Note in Obsidian
<img width="241" alt="image" src="https://github.com/ObsidianToAnki/Obsidian_to_Anki/assets/54943469/4b3c6e75-58c9-469f-baed-af74b85960ff">

Note in Anki after adding audio
![image](https://github.com/ObsidianToAnki/Obsidian_to_Anki/assets/54943469/03ffbced-5213-4a1c-b49f-c503e21b2172)

Note in Anki after using "scan vault" with Obsidian_to_Anki
![image](https://github.com/ObsidianToAnki/Obsidian_to_Anki/assets/54943469/c3a4a498-e8fb-4d9f-ab77-cec7ab73bc07)

**Comment by ShootingKing-AM** [comment]
True, there is only one side communication as of now. That is Obsidian to Anki.

**Comment by anahodil** [comment]
Sure, I wasn’t expecting it to populate the fields and obsidian. I was just hoping it wouldn’t replace the fields in Anki, unless specifically called out in obsidian.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by samawelch** [comment]
I'm also having this problem. It would be more convenient if fields we don't specify in the Obsidian file aren't overwritten. A partial fix is to break out flashcards into separate .md files, as I believe this behaviour only occurs when files are modified.
